CHANGELOG.md

```diff
@@ -3,7 +3,8 @@ A more detailed list of changes is available in the corresponding milestones for
 
 
 ## 0.8.0 (2020-Jun-??)
-  - ...
+### Changes to existing checks
+  - **[com.google.fonts/check/varfont/unsupported_axes]**: Removed opsz axis and added slnt axis
```

Comment:
please always mention the issue number on the CHANGELOG.md entries. I can add this one later, but please include it on your future PRs